staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -18,6 +18,18 @@ package v1beta1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// ConversionStrategyType describes different conversion types.
+type ConversionStrategyType string
+
+const (
+	// NoneConverter is a converter that only sets apiversion of the CR and leave everything else unchanged.
+	NoneConverter ConversionStrategyType = "None"
```

Comment:
the Godoc shows up in api docs later. Wondering whether you should refer to the string literal `None` instead. Please compare with constants in other types.go files.

Reply:
Example:https://github.com/kubernetes/kubernetes/blob/ae2edc439e54fef7182b9e9c51aa7369cb32d567/pkg/apis/apps/types.go#L54
Look like the name is used not the string value.